dev/statement_serializer.h

```diff
@@ -886,6 +886,26 @@ namespace sqlite_orm {
             }
         };
 
+        template<>
+        struct statement_serializer<null_t, void> {
+            using statement_type = null_t;
+
+            template<class Ctx>
+            std::string operator()(const statement_type& statement, const Ctx& context) const {
+                return "NULL";
```

Comment:
I think we should remove `std::string` cast operators from AST nodes cause they don't help us but just spread serialization between `statement_serializer` and node's type. That's why I added the whole serialization literals right here